magento/pwa-studio
# Hide Accordion closed Sections instead of unmounting them

## Description

This PR hides accordion section contents that aren't open instead of not rendering them at all.
This saves components from mounting / unmounting on section open changes, but may overfetch (if a section is never opened, its contents may still make network calls).

This PR also sets all price adjustment accordion sections to closed initially, which is the intended UX.

## Related Issue





Closes PWA-360.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the `/cart` page
1. Inspect element on the price adjustments `Accordion`
1. See that the contents of the sections that are closed still appear in the DOM, they're just hidden from the user

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
